pkg/kube/client.go

```diff
@@ -461,6 +461,7 @@ func (c *client) RunAndWait(stop <-chan struct{}) {
 	c.metadataInformer.Start(stop)
 	c.istioInformer.Start(stop)
 	c.serviceapisInformers.Start(stop)
+	c.memberRoll.Start(stop)
```

Comment:
I'm not sure if this is correct, but it seemed the member roll controller needed to be started somewhere.

Reply:
Oops. Yeah, but we need to check if it is `nil` before starting it.